ccl/workloadccl: BenchmarkImportFixture fails with memory budget exceeded

```
--- FAIL: BenchmarkImportFixture/tpcc/warehouses=1
    bench_test.go:42: 
        	Error Trace:	/mnt/data1/microbench/pkg/ccl/workloadccl/bin/bench_test.go:42
        	            				/mnt/data1/microbench/pkg/ccl/workloadccl/bin/bench_test.go:55
        	            				/mnt/data1/microbench/pkg/ccl/workloadccl/bin/benchmark.go:193
        	            				/mnt/data1/microbench/pkg/ccl/workloadccl/bin/benchmark.go:233
        	            				/mnt/data1/microbench/pkg/ccl/workloadccl/bin/asm_amd64.s:1594
        	Error:      	Received unexpected error:
        	            	importing table history: pq: not enough memory available to create a BulkAdder: root: memory budget exceeded: 33554432 bytes requested, 103008256 currently allocated, 134217728 bytes in budget
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportFixture.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:404
        	            	  | github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:168
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:75
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (2) importing table history
        	            	Wraps: (3) pq: not enough memory available to create a BulkAdder: root: memory budget exceeded: 33554432 bytes requested, 103008256 currently allocated, 134217728 bytes in budget
        	            	Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *pq.Error
        	Test:       	BenchmarkImportFixture/tpcc/warehouses=1
--- FAIL: BenchmarkImportFixture
FAIL
```

Note:
```
pq: not enough memory available to create a BulkAdder: root: memory budget exceeded: 33554432
```

To reproduce:
```
./dev bench pkg/ccl/workloadccl -f=BenchmarkImportFixture --ignore-cache --bench-time=1s --verbose --stream-output --timeout 120m
```

Jira issue: CRDB-23895

--- a/pkg/ccl/workloadccl/bench_test.go
+++ b/pkg/ccl/workloadccl/bench_test.go
@@ -18,18 +18,26 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/testutils/serverutils"
 	"github.com/cockroachdb/cockroach/pkg/testutils/skip"
 	"github.com/cockroachdb/cockroach/pkg/testutils/sqlutils"
+	"github.com/cockroachdb/cockroach/pkg/util/log"
 	"github.com/cockroachdb/cockroach/pkg/workload"
 	"github.com/cockroachdb/cockroach/pkg/workload/tpcc"
 	"github.com/stretchr/testify/require"
 )
 
 func benchmarkImportFixture(b *testing.B, gen workload.Generator) {
+	defer log.Scope(b).Close(b)
 	ctx := context.Background()
 
 	var bytes int64
 	b.StopTimer()
 	for i := 0; i < b.N; i++ {
-		s, db, _ := serverutils.StartServer(b, base.TestServerArgs{UseDatabase: `d`})
+		s, db, _ := serverutils.StartServer(
+			b,
+			base.TestServerArgs{
+				UseDatabase:       `d`,
+				SQLMemoryPoolSize: 1 << 30, /* 1GiB */ // default 128MiB might be insufficient
+			},
+		)
 		sqlDB := sqlutils.MakeSQLRunner(db)
 		sqlDB.Exec(b, `CREATE DATABASE d`)
 

